symplyphysics/laws/thermodynamics/equations_of_state/van_der_waals/critical_pressure.py

```diff
@@ -19,7 +19,7 @@
     symbols,
 )
 
-critical_pressure = clone_as_symbol(symbols.pressure, subscript="\\text{c}")
+critical_pressure = clone_as_symbol(symbols.pressure, display_symbol="p_c", display_latex="p_\\text{c}")
```

Comment:
AFAIR you've added regex to the printer to convert display_latex to code. I suggest to remove it eventually.